src/qml/components/SelectContactsList.qml

```diff
@@ -0,0 +1,216 @@
+        duration: Theme.animationDuration
+        easing.type: Easing.InExpo
+    }
+
+    Component {
+        id: contactListHeader
+
+        Item {
+            width: parent.width
+            height: d.headerHeight
+            opacity: d.headerOpacity
+            enabled: modelListView.state === "show header"
+
+            ListDelegate {
+                id: contactListDelegate
+                anchors.centerIn: parent
+                width: parent.width
+                height: d.defaultChatHeight
+
+                Avatar {
+                    id: avatar
+                    nickname: modelListView.newContactText
+                    Layout.alignment: Qt.AlignVCenter
+                }
+
+                Column {
+                    Layout.fillWidth: true
+                    clip: true
+
+                    Text {
+                        color: Theme.primaryTextColor
+                        font.pointSize: UiHelper.fixFontSz(15)
+                        text: modelListView.newContactText
+                    }
+
+                    Text {
+                        color: Theme.secondaryTextColor
+                        font.pointSize: UiHelper.fixFontSz(12)
+                        text: qsTr("Click here to select contact")
+                        width: parent.width
+                        elide: Text.ElideRight
+                        textFormat: Text.RichText
+                    }
+                }
+
+                onClicked: {
+                    if (d.model.selection.multiSelect) {
+                        d.model.toggleById(modelListView.newContactText)
+                    }
+                    contactSelected(modelListView.newContactText)
+                }
+            }
+        }
+    }
+
+    Component {
+        id: contactListComponent
+
+        ListDelegate {
+            id: contactListDelegate
+            leftInset: 0
+            rightInset: 0
+            width: modelListView.width
+            height: d.defaultChatHeight
+
+            Item { // we need this item!
+                width: parent.width
+                height: parent.height
+
+                Row {
+                    width: parent.width
+                    height: parent.height
+                    anchors.verticalCenter: parent.verticalCenter
+
+                    spacing: Theme.smallSpacing
+
+                    add: Transition {
+                        NumberAnimation { property: "scale"; from: 0.9; to: 1; duration: Theme.animationDuration; easing.type: Easing.InOutCubic }
+                        NumberAnimation { property: "opacity"; from: 0; to: 1; duration: Theme.animationDuration; easing.type: Easing.InOutCubic }
+                    }
+
+                    move: Transition {
+                        NumberAnimation { properties: "x,y"; duration: Theme.animationDuration; easing.type: Easing.InOutCubic }
+                    }
+
+                    // TODO(fpohtmeh): replace with image
+                    Item {
```

Comment:
This will definitely work without wrapping into Item

Reply:
Yes, this would work.